modules/core/src/main/java/org/apache/ignite/logger/java/JavaLogger.java

```diff
@@ -368,14 +372,27 @@ public void setWorkDirectory(String workDir) {
     @Override public void setNodeId(UUID nodeId) {
         A.notNull(nodeId, "nodeId");
 
-        if (this.nodeId != null)
+        postfix(nodeId, U.id8(nodeId));
+    }
+
+    /** {@inheritDoc} */
+    @Override public void setPostfix(String postfix) {
```

Comment:
always used to set node it. 
any reason to have this method, field, class, etc?